exif.cpp

```diff
@@ -706,6 +706,11 @@ int easyexif::EXIFInfo::parseFromEXIFSegment(const unsigned char *buf,
           }
           break;
 
+        case 0xa210:
+            // EXIF Focal plane resolution unit
+            this->LensInfo.FocalPlaneResolutionUnit = result.val_short().front();
```

Comment:
We need to check that the read data is in the appropriate format (a short), else it can lead to a bad cast
```
if (result.format() == 3 && result.val_short().size()) {
   this->LensInfo.FocalPlaneResolutionUnit = result.val_short().front();
}```